constraint/config/crds/externaldata.gatekeeper.sh_providers.yaml

```diff
@@ -55,9 +54,39 @@ spec:
         type: object
     served: true
     storage: true
-status:
-  acceptedNames:
-    kind: ""
-    plural: ""
-  conditions: []
-  storedVersions: []
+  - name: v1beta1
+    schema:
+      openAPIV3Schema:
+        description: Provider is the Schema for the providers API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: Spec defines the Provider specifications.
+            properties:
+              caBundle:
+                description: CABundle is a base64-encoded string that contains the
+                  TLS CA bundle in PEM format. It is used to verify the signature
+                  of the provider's certificate.
+                type: string
+              timeout:
+                description: Timeout is the timeout when querying the provider.
+                type: integer
+              url:
+                description: URL is the url for the provider. URL is prefixed with
+                  http:// or https://.
+                type: string
+            type: object
+        type: object
+    served: true
+    storage: false
```

Comment:
shouldn't storage be true here and false for v1alpha1?

Reply:
I discussed this with @maxsmythe. for backwards compat, we thought about leaving it in v1alpha1 for now

Reply:
Can you please explain or link me to the rationale for this?

Reply:
If a user rolls back G8r (including the ExternalData CRD), the stored data can continue to be served otherwise, a CRD that only knows v1alpha1 may break if it reads a different storage version. No other benefit AFAIK.

Reply:
Gotcha. When is it safe for us to change the beta storage to true? I don't recall doing this for other GK CRDs moving from alpha to beta. Should we follow the same suit in the future when we update the API for all GK CRDs? Or is the ExternalData CRD unique?

Reply:
This is the first time we've done it. Probably not a bad idea to keep doing it moving forward.

As to when we'd shift... I guess when we have one stable version out that has v1beta1 in it so rolling back the CRD doesn't break things?

Reply:
Sounds good! created https://github.com/open-policy-agent/gatekeeper/issues/2457 to track it for v3.12